MsCorePkg/MsCorePkg.dec

```diff
@@ -136,6 +136,26 @@
   ## Default: 1024 * 4KiB = 4MB
   gMsCorePkgTokenSpaceGuid.PcdDebugFileLoggerAllocatedPages|1024|UINT32|0x4000001C
 
+  ## Pcd value representing the Pk for a platform
+  # Empty by default

```

Comment:
Since this is reference, should the comment explain why it's "Empty by default"

Reply:
I've updated the comments to be more specific!